src/index.js

```diff
@@ -281,10 +296,17 @@ function cssVars(options = {}) {
             if (settings.updateDOM) {
                 const targetElm = settings.rootElement.host || (settings.rootElement === document ? document.documentElement : settings.rootElement);
 
+                let hasVarChange = false;
+
                 // Set variables using native methods
                 Object.keys(settings.variables).forEach(key => {
-                    targetElm.style.setProperty(key, settings.variables[key]);
+                    const varValue = settings.variables[key];
+
+                    hasVarChange = hasVarChange || varValue !== getComputedStyle(targetElm).getPropertyValue(key);
+                    targetElm.style.setProperty(key, varValue);
                 });
+
+                handleFinally(hasVarChange);
```

Comment:
I think this should probably be outside of the `if` statement--shouldn't `onFinally` be called here even when `settings.updateDOM` is false?

Reply:
Probably: See https://github.com/jhildenbiddle/css-vars-ponyfill/pull/119/files/327a6b9502d4e5834987cb55f25362432404479e#r411533716 for discussion.